src/support/logos.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: "layouts/default.liquid"
+title: Logos
+meta_description: "The logos for us and our tools"
+---
+You're free to use our logos without request, so here are the files!
+
+# Usage guidelines
+
+We ask that you:
+
+* Do use alt-text whenever possible, so we've provided examples.
+* Don't adjust the colors used
+* Don't adjust the fonts used
+* Don't put any within on 'dots' width of the logo
+
+# OA.Works
+
+Black & White: SVG | PNG | JPG
```

Comment:
Avoid using the pipe symbol as separator -- it doesn’t look great. Can display them as buttons by adding the `.button` class.